makeinvoice.go

```diff
@@ -80,6 +82,15 @@ type EclairParams struct {
 func (l EclairParams) getCert() string { return l.Cert }
 func (l EclairParams) isTor() bool     { return strings.Index(l.Host, ".onion") != -1 }
 
+type StrikeParams struct {
+	Cert string
+	Host string
```

Comment:
wait so strikeparams can skip implementing these and still be ok to be defined as a `BackendParams` ?

Reply:
yes, it is defined as `BackendParams` on account of the functions only, not the properties.
we should probably remove those functions too, since they're stupid and useless.

Reply:
I just returned empty and false, does that help?

Reply:
yes